src/context.js

```diff
@@ -8,20 +8,21 @@ class Context {
     walletId,
     apiKey,
     passphraseGetter = noopPassphraseGetter,
-    host = '127.0.0.1'
+    host = '127.0.0.1',
+    optionalNodeApiKey,
```

Comment:
Please rename this parameter to `nodeApiKey`, then this LGTM.

Reply:
updated